inference-engine/include/ie_precision.hpp

```diff
@@ -254,7 +249,7 @@ class Precision {
         if (precisionInfo.bitsSize == 0) {
             IE_THROW() << " cannot estimate element if precision is " << precisionInfo.name;
         }
-        return precisionInfo.bitsSize >> 3;
+        return precisionInfo.bitsSize >= 8 ? precisionInfo.bitsSize >> 3 : 1;
```

Comment:
1.  We have inconsistent description
  ```
/**
 * @brief Returns size of single element of that precision in **bits**
 * @returns Number of **bytes** per element
 */
```
2. The formula seems to be wrong for cases when `bitsSize > 8 && bitsSize % 8 != 0`. For now it's not that critical, but we may have some issues in the future with precisions like TF32 (19 bits)

Reply:
fixed